src/applications/vaos/new-appointment/components/ProviderSelectPage/SchedulingInfo.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export default function SchedulingInfo() {
```

Comment:
Hmm this doesn't seem like it's being used yet so do we need to include it in this PR?

Reply:
bump?